### PR TITLE
HOTFIX: reduce group size by removing duplicates

### PR DESCRIPTION
**Pull request by DevFojo**
## Test

> Service: https://telerivet.com/p/0c6396c9/service/SVea040ce197886f68/edit

1. Dial in short code `*689#`
2. Enter account number `12336313`
3. Select menu option `11` to go to group repayments
4. Enter last 4 digits of national ID `4219`
5. The returned group repayments have no duplicate member.

### CODE REVIEW

**Review by Bahatiroben — DISMISSED** [review]

**Review by nyabongo — CHANGES_REQUESTED** [review]
Could you add a test case where duplicates are removed

File: group-repayments/inputHandlers/lastFourIdDigitsHandler.test.js [hunk 1]
```diff
@@ -30,6 +30,83 @@ describe('Last four nid digits input handler', () => {
...
+        '2) james bond: 5700 RwF\n' +
+        '44) Go back');
+        expect(promptDigits).toHaveBeenCalledWith('view_individual_balance_menu', {
+            submitOnHash: false,
+            maxDigits: 2,
+            timeout: 5
+        });
+    });
+    
+    it('should merge duplicate members', () => {
```
Comment (nyabongo) [inline group-repayments/inputHandlers/lastFourIdDigitsHandler.test.js:69]: The test titles should be distinct from each other, this one is the same as the previous one

**Review by nyabongo — CHANGES_REQUESTED** [review]

**Review by nyabongo — APPROVED** [review]

**Review by Bahatiroben — APPROVED** [review]
